frontstage/controllers/iac_controller.py

```diff
@@ -22,14 +22,14 @@ def get_iac_from_enrolment(enrolment_code, validate=False):
         if response.status_code == 404:
             logger.info('IAC not found')
             if validate:
-                raise ApiError(logger.info, response, message='Invalid enrolment code used')
+                raise ApiError(logger, response, log_level='info', message='Invalid enrolment code used')
```

Comment:
This seems wrong. It feels like log_level shouldn't be a part of the exception. It should be a new type of Exception (that could extend APiError)

Reply:
@benjefferies perhaps but not relevant to this PR. It was refactored this way some time ago when we deprecated frontstage-api.

Reply:
@benjefferies @jcox-dev Improvements team have raised a ticket to refactor logging and specifically this 👍

Reply:
this was just to fix a bug introduced

Reply:
The log level isn't being used, when the ApiError is handled in error_handlers.py it doesn't use the logger passed via the exception, it just uses uses `logger.error`. I think we should be discussing why we generate a 500 error when an enrolment code is not found or is not active.

Reply:
That's a valid point @ajmaddaford, in terms of this piece of work, how should I look to rectify this.

Reply:
I think we should continue to log at error as we're returning a 500.